amundsen_application/static/js/components/TableDetail/ColumnListItem/index.spec.tsx

```diff
@@ -124,27 +124,137 @@ describe('ColumnListItem', () => {
+        const { props, wrapper } = setup({
+          editText: '',
+          editUrl: '',
+        });
+        const instance = wrapper.instance();
+        instance.setState({ isExpanded: true });
+        const newWrapper = shallow(instance.render());
+        expect(newWrapper.find(EditableSection).exists()).toBe(true);
+        const editableSection = newWrapper.find(EditableSection);
+        expect(editableSection.props()).toMatchObject({
+          title: 'Description',
+          readOnly: !props.data.is_editable,
+          editText: props.editText,
+          editUrl: props.editUrl,
+        });
+      });
+
+      describe('when empty description', () => {
+        it('renders EditableSection with empty edit text and url for editable column description when expanded', () => {
+          const { props, wrapper } = setup({
+            data: {
+              name: 'test_column_name',
+              description: '',
+              is_editable: true,
+              col_type: 'varchar(32)',
+              stats: [
+                {
+                  end_epoch: 1571616000,
+                  start_epoch: 1571616000,
+                  stat_type: 'count',
+                  stat_val: '12345',
+                },
+              ],
+            },
+            editText: '',
+            editUrl: '',
+          });
+          const instance = wrapper.instance();
+          instance.setState({ isExpanded: true });
+          const newWrapper = shallow(instance.render());
+          expect(newWrapper.find(EditableSection).exists()).toBe(true);
+          const editableSection = newWrapper.find(EditableSection);
+          expect(editableSection.props()).toMatchObject({
+            title: 'Description',
+            readOnly: !props.data.is_editable,
+            editText: props.editText,
+            editUrl: props.editUrl,
+          });
+        });
+
+        it('does not render EditableSection with empty edit text and url for non-editable column description when expanded', () => {
+          const { wrapper } = setup({
+            data: {
+              name: 'test_column_name',
+              description: '',
+              is_editable: false,
+              col_type: 'varchar(32)',
+              stats: [
+                {
+                  end_epoch: 1571616000,
+                  start_epoch: 1571616000,
+                  stat_type: 'count',
+                  stat_val: '12345',
+                },
+              ],
+            },
+            editText: '',
+            editUrl: '',
+          });
+          const instance = wrapper.instance();
+          instance.setState({ isExpanded: true });
+          const newWrapper = shallow(instance.render());
+          expect(newWrapper.find(EditableSection).exists()).toBe(false);
+        });
+
+        it('renders EditableSection with non-empty edit text and url for non-editable column description when expanded', () => {
```

Comment:
I guess this one shouldn't be nested inside the "when empty description" describe

Reply:
In this case also we have empty column description, but user have option to edit and add some description, hence I have put it inside "when empty description" describe.